src/index.js

```diff
@@ -6,4 +6,5 @@ export {queries}
 
 export * from './queries'
 export * from './wait'
+export * from './waitForElement'
```

Comment:
This is a bit of a nit-pick and I really should have an eslint rule for this, but due to issues with file casing + different operating systems + git, I prefer filenames to always use kebab-case rather than camelCase. Could you update the files to use kebab-case please?

Reply:
Okay, but tell this to React team. It's for the sake of searchability that the file is literally named the same as the thing it contains and exports. Better have a lint rule that ensures the names strictly match case with the filesystem (and there is such, I think in the eslint-plugin-import).

Reply:
I have told them and they rejected it. But I've had too many people bitten by file casing issues across operating systems to do anything else.

Reply:
If searchability is key then you say: "Oh, here's a thing called `waitForElement` in the code, so I'll turn that to kebab-case and search for `wait-for-element`." Believe me, I've felt these pains before as well. We just have different solutions.

In addition, it's why I'm starting to avoid `default export`s now because the greppability of a named export is better.

Reply:
I just select the thing, cmd+shift+f, no need to convert. Especially when doing large refactorings. Default exports I agree can become messy if coders don't follow to name them the same at import site, it's a matter of discipline which is not enforced by the program structure.

Reply:
👉  https://github.com/kentcdodds/dom-testing-library/pull/10/commits/1252d0824d454b27ff8ed92390af681984069668